concourse/pipeline.yml

```diff
@@ -151,6 +156,40 @@ jobs:
         file: git-master/concourse/tasks/smoke-test.yml
         params:
           URL: 'https://coronavirus-shielding-support.service.gov.uk/applying-on-own-behalf'
-          MESSAGE: "Checks that the application deployed to staging is not serving HTTP error codes"
+          MESSAGE: "Checks that the application deployed to production is not serving HTTP error codes"
         on_failure: *slack_alert_on_failure
         on_success: *slack_alert_on_success
+
+  - name: continuous-smoke-test-staging
+    plan:
+      - get: every-10m
+        trigger: true
+      - get: git-master
+      - task: smoke-test
+        file: git-master/concourse/tasks/smoke-test.yml
+        params:
+          WEB_APP_BASE_URL: 'https://gds-shielded-vulnerable-people-service-staging.london.cloudapps.digital/applying-on-own-behalf'
+          MESSAGE: "Checks that the application deployed to staging is not serving HTTP error codes"
+        on_failure: *slack_alert_on_failure
+      - task: cronitor-heartbeat
+        timeout: 1m
+        params:
+          CRONITOR_URL: ((svp-form/cronitor-heartbeat-stg))
+        file: git-master/concourse/tasks/cronitor.yml
+
+  - name: continuous-smoke-test-prod
+    plan:
+      - get: every-10m
+        trigger: true
+      - get: git-master
+      - task: smoke-test
+        file: git-master/concourse/tasks/smoke-test.yml
+        params:
+          WEB_APP_BASE_URL: 'https://coronavirus-shielding-support.service.gov.uk/applying-on-own-behalf'
+          MESSAGE: "Checks that the application deployed to production is not serving HTTP error codes"
+        on_failure: *slack_alert_on_failure
+      - task: cronitor-heartbeat
+        timeout: 1m
+        params:
+          CRONITOR_URL: ((svp-form/cronitor-heartbeat-prod))
```

Comment:
guess this variable has been added to concourse?

Reply:
Yup, I've got a cronitor account and the URL you have to ping looks like https://cronitor.link/some-secret-key so i wanted to make sure it wasn't public.

Reply:
fantastic